### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by deniodev**
Good day.

Kindly see the desktop version of my portfolio using media queries.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,145 +42,181 @@ <h2 class="subtitle">I'm a Software Developer</h2>
...
-						<ul class="cards-btn">
-							<li class="html-btn"><button class="project-btn">HTML</button></li>
-							<li><button class="project-btn bootstrap">Bootstrap</button></li>
-							<li class="rubi-btn"><button class="project-btn">Ruby</button></li>
-						</ul>
-					</div>
-					<button class="project-see-btn">See Project</button></a>
-				</div> 
-            </div>
+            </div>
```
Comment (didierganthier) [inline index.html:79]: Nice having a featured section 👍🏽 but make sure to adjust the spacing and the sizes of the elements, it's not looking quite right yet on several screen sizes 👀  take a look at the differences:

## Your design

Screenshot 2023-03-31 at 09 06 14

## Figma

Screenshot 2023-03-31 at 09 06 35

As you can notice, the featured image 🖼️ should be taking much more space (60% of the width) and the rest should be taking (40%) while being aligned to the left ◀️, make sure to adjust that along with the spacing between the elements such as the list of technologies 🤔 also the heading of the featured project should come first and the font size should be bigger than the following text in the section. 🤓

File: index.html [hunk 2]
```diff
@@ -35,145 +42,181 @@ <h2 class="subtitle">I'm a Software Developer</h2>
...
+                  <div>
+                      <ul class="cards-btn">
+                        <li class="html-btn"><button class="project-btn">HTML</button></li>
+                        <li><button class="project-btn bootstrap b-btn">Bootstrap</button></li>
+                        <li class="rubi-btn"><button class="project-btn ruby">Ruby</button></li>
+                      </ul>
+                  </div>
+                </div>
+                <a href="#" class="first"><button class="project-see-btn first">See Project</button></a>              
+              </div>
```
Comment (didierganthier) [inline index.html:95]: The cards look nice 💯 but they need to be spaced better, the container of your cards has too much spacing or maybe your cards don't take enough space 🤔 this is what you're supposed to have 👇🏽 

Screenshot 2023-03-31 at 09 21 35

This is what you have on full-screen size 👇🏽 

Screenshot 2023-03-31 at 09 23 00

Make sure to also adjust the spacing of the elements inside of each card right now 👁️ last thing but not least, I also noticed that you designed the first one differently (just like it looks in the Figma )

Screenshot 2023-03-31 at 09 31 20

 but the goal of the Figma when making the first design different was to showcase how the card should look when the mouse goes over it (hover effect) 🤓 so keep that design in mind and just adjust it to make all cards look like that but only when they're hovered. If you have any questions about this or if my explanation is a bit confusing, feel free to tag me. 😉

File: index.html [hunk 3]
```diff
@@ -209,25 +252,29 @@ <h2 class="a-title">About me</h2>
...
+                <li><a href="https://www.linkedin.com/in/denionhanale/"><img src="./images/Icon-Linkedin.png" alt="linkedin"></a></li>
                 <li><a href="#"><img src="./images/Icon-Angelist.png" alt="angelist"></a></li>
-                <li><a href="#"><img src="./images/Icon-Twitter.png" alt="twitter"></a></li>
+                <li><a href="https://twitter.com/DNhanale"><img src="./images/Icon-Twitter.png" alt="twitter"></a></li>
                 <li><a href="#"><img src="./images/Icon-Medium.png" alt="Medium"></a></li>
-            </ul>
-        </div>
-         </footer>         
+          </ul>
+        </footer>
```
Comment (didierganthier) [inline index.html:277]: Good looking social icons 😻 I love how aligned they are but I feel like their container is too big at the moment, it makes them look too small. Take a look at what the Figma has 👇🏽 

Screenshot 2023-03-31 at 09 40 31

And this is what you have so far 👇🏽 

Screenshot 2023-03-31 at 09 40 52

Did I lie? 🤔 I didn't think so 😄 and I believe you can easily adjust that 💪🏽

File: index.html [hunk 4]
```diff
@@ -35,145 +42,181 @@ <h2 class="subtitle">I'm a Software Developer</h2>
...
 
         <section class="aboutme">
-          <div class="a-header">
+          <div>
             <h2 class="a-title">About me</h2>
             <p class="a-text">
                 Hello I'm a software developer! I can help you build a product , feature or website Look
                 through some of my work and experience! If you like what you see and have a project you need coded, don't
                 hestiate to contact me.  
             </p>
```
Comment (didierganthier) [inline index.html:211]: It's good learning a bit about you 🤝 but to make this section even more enjoyable to read, after the Section's Title, also aligned to the left there should be 4 to 5 lines of text describing your experience. Your text is only taking two lines for now on full-screen size, you could perhaps adjust that by trapping it inside a box/container. 🤔  Take a look 👇🏽 

Screenshot 2023-03-31 at 09 46 19

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @deniodev ,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Comment by deniodev** [comment]
Hi @didierganthier 
Good day

Kindly see the requested changes done:
- Adjust the spacing and the sizes of the works section;
- Adjust the spacing and the sizes of the cards section;
- Add  hover effects on the cards section;
- Align the titles and icons on the skills section;
- Adjust the social media icons;
- Adjust the text in the About-me section;

**Review by amon-cofie — APPROVED** [review]
Hi @deniodev , 👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Descriptive PR 🚀 
- Linters are passing 🚀 
- GitHub flow used properly 🚀 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**